views/confirm-company.html

```diff
@@ -86,13 +86,13 @@ <h2 class="govuk-heading-l">{{company.companyName}}</h2>
         {{ govukButton({
           attributes: {
             id: "submit",
-            "data-event-id": "confirm-and-continue-button"
+            "data-event-id": "confirm this is the correct company page - confirm and continue button"
           },
           text: "Confirm and continue"
         }) }}
 
         <p>
-          <a href="/officer-filing-web/company-number" data-event-id="choose-a-different-company-link">Choose a different company</a>
+          <a href="/officer-filing-web/company-number" data-event-id="confirm this is the correct company page - Change Company Link">Choose a different company</a>
```

Comment:
can we revert these Matomo ids back as they should have the dash's and we don't want the names too long.  Cheers.